### ISSUE TITLE
Styles breaking in Obsidian 1.37.

### ISSUE THREAD

**Issue by zhouxinghong** [KosmosisDire/obsidian-webpage-export#178]
Hello! This extension is completely unavailable in Obsidian 1.37.

**Comment by KosmosisDire** [comment]
Hello, do you mean 1.3.7? I am on 1.3.7 and can see it in the community plugins. Can you elaborate on what the problem is? Thank you

**Comment by zhouxinghong** [comment]
I mean: obsidian version 1.37

**Comment by zhouxinghong** [comment]
I use extension 1.6 or 1.5.2, etc. After exporting in Obsidian 1.37, the styles are all messed up.

**Comment by KosmosisDire** [comment]
Oh so you are able to download the extention, I thought you meant that you could not install it. You will have to be much more specific about exactly what the problem is, maybe show some screenshots of what it looks like.

**Comment by KosmosisDire** [comment]
Could you also run the "Show Debug Info" command on your obsidian vault and copy paste the output.
![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/39423700/8d68fe46-a6ea-4b17-8586-f1c64f2e8a7f)

**Comment by R4wd0g** [comment]
Hello. Same situation here after update the plugin to latest version (1.6.0)

![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/29448208/6e8a4ebf-0eca-4fba-88a7-1db20f353cd5)

it occurs to every .md page i try to export. things i've made:

- remove accentuation from file names
- unmark some options before export (css, file tree, etc)
- tried to export the first .md file on the root folder.
- tested on another network

Do you have any idea about what can be happening?
thanks bro

**Comment by KosmosisDire** [comment]
@R4wd0g Hi this does look very messed up, is this screenshot from inside of obsidian? Or is this actually what the exported .html file looks like? 

Can you open the console in obsidian (Ctrl+Shift+I) and then export again, and paste a screenshot of the output from that console to here? Thanks!

**Comment by R4wd0g** [comment]
> @R4wd0g Hi this does look very messed up, is this screenshot from inside of obsidian? Or is this actually what the exported .html file looks like?

Yes. The screenshot is from inside of Obsidian. The final .html exported appears like this:
![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/29448208/3e2a8858-04ef-4728-95ee-1d69a7abaeb8)



> Can you open the console in obsidian (Ctrl+Shift+I) and then export again, and paste a screenshot of the output from that console to here? Thanks!

Sure:

![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/29448208/dec2f22b-6fc5-48a2-8fad-8a4b9f848d35)

When i click on the reference:

![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/29448208/502e91e5-c488-4084-91ee-0019f322a14e)

**Comment by KosmosisDire** [comment]
This is fixed in v1.7.0

**Comment by R4wd0g** [comment]
thanks for your effort in these new versions.
now, when i click on export, the final result appears to be the same. (no page render on browser)

here is a message from log:
![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/29448208/4e3ba307-cfc3-4264-9d02-6e0a4aa0fc2e)

I have disabled some plugins intending to find a culprit, but no lucks. I've disabled custom icon css too.

**Comment by KosmosisDire** [comment]
> Could you also run the "Show Debug Info" command on your obsidian vault and copy paste the output.
> ![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/39423700/8d68fe46-a6ea-4b17-8586-f1c64f2e8a7f)
> 

@R4wd0g Hi thank you for the update, could you follow the instructions in this comment for me? That would be very helpful, thanks!

**Comment by R4wd0g** [comment]
Hello @KosmosisDire,
I've disabled the plugin called **Opener** and it worked back.
So, it appears that Opener plugin doens't seems compatible with your plugin :'(.
But luckly the solution was found and i can disable the Opener plugin ever when i need to export some .md as HTML.
Thanks for your support.

**Comment by KosmosisDire** [comment]
Huh very interesting, I am glad you found the solution? I'll look into it and see if there is a way I can get them to work together
